commcare_connect/opportunity/models.py

```diff
@@ -198,6 +198,18 @@ def budget_per_user(self):
     def is_active(self):
         return bool(self.active and self.end_date and self.end_date >= now().date())
 
+    @property
+    def program_name(self):
+        return self.managedopportunity.program.name if self.managed else None
+
+    @property
+    def current_status(self):
+        if self.is_active:
+            return "Active"
+        if self.end_date and self.end_date < now().date():
+            return "Ended"
+        return "Inactive"
```

Comment:
It's better to introduce a `has_ended` attr on the model and use it in the template instead of introducing three new undeclared constants. You will template will become

```
{% if object.is_active %}
    <span class="badge badge-md positive-dark">Active</span>
{% elif object.has_ended %}
    <span class="badge badge-md warning-dark">Ended</span>
{% else %}
    <span class="badge badge-md negative-dark">Inactive</span>
{% endif %}
```